datashuttle/command_line_interface.py

```diff
@@ -326,6 +291,7 @@ def make_sub_dir(project, args):
     metavar="",
 )
 make_sub_dir_parser.add_argument(
+    "--experiment-type",
     "--experiment_type",
```

Comment:
Why is this specified twice? Same for the two fucntion calls above this one.

Reply:
Thanks for this, just realised the parser do not accept aliases :(

```
    make_config_file_parser.add_argument(
        "--show-transfer-progress",
        aliases=["--show_transfer_progress"],
        required=False,
        action="store_true",
        help=help("flag_default_false"),
    )
```
```
(datashuttle_conda) C:\Users\Joe\work\git-repos\datashuttle>datashuttle test update_config remote-path hello world
Traceback (most recent call last):
  File "C:\Users\Joe\miniconda3\envs\datashuttle_conda\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\Joe\miniconda3\envs\datashuttle_conda\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\Joe\miniconda3\envs\datashuttle_conda\Scripts\datashuttle.exe\__main__.py", line 4, in <module>
  File "C:\Users\Joe\work\git-repos\datashuttle\datashuttle\command_line_interface.py", line 836, in <module>
    parser = construct_parser()
  File "C:\Users\Joe\work\git-repos\datashuttle\datashuttle\command_line_interface.py", line 420, in construct_parser
    make_config_file_parser.add_argument(
  File "C:\Users\Joe\miniconda3\envs\datashuttle_conda\lib\argparse.py", line 1423, in add_argument
    action = action_class(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'aliases'
```